### ISSUE TITLE
Support a variable number of rows

### ISSUE THREAD

**Issue by aaronpowell** [Sese-Schneider/ha-energy-overview-card#3]
Presently the card supports three energy inputs, but I was wondering if it's possible to allow a variable number of them, so you could have one or ten.

I'd anticipate that the changes would be to make the `EnergyOverviewConfig` type have an array of entity fields:

```ts
export interface EnergyOverviewConfig {
  type: string;

  entities: {
    current: string;
    power_factor: string;
    voltage: string;
    power: string;
  }[];
}
```

And then card `render` method would iterate over the `config.entities` array to read each entity set to build up the existing `data` array (there'd probably have to be something else done to make the colour of each row support more).

**Comment by Sese-Schneider** [comment]
@aaronpowell thank you for your suggestion! I've been actively working on changing this for the next version.

**Comment by Sese-Schneider** [comment]
@aaronpowell the change has been implemented in the newest release `2.0.0`
